api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -224,7 +224,7 @@ public function deleteOne($id, $childId)
         $model = $this->findParentEntity($id);
         $childModel = $this->findOrFailChildEntity($childId, $model);
 
-        $childModel->delete();
+        $model->deleteChild($childModel, $this->relationName);
```

Comment:
@xiphiaz Alright, I think the new updates are more in line with clarity of the code's responsibilities regarding child deletion, so please review and see what you think.